components/backup/src/endpoint.rs

```diff
@@ -293,33 +389,35 @@ impl BackupRange {
                 return Err(e);
             }
         }
+        drop(snap_store);
+        let stat = scanner.take_statistics();
+        if start_scan.saturating_elapsed_secs() > 30.0 {
```

Comment:
place `start_scan.saturating_elapsed_secs()` into a variable. the elapsed duration is not a pure function.